app/models/container.rb

```diff
@@ -18,9 +19,6 @@ class Container < ApplicationRecord
   has_many   :metric_rollups, :as => :resource
   has_many   :vim_performance_states, :as => :resource
 
-  # Needed for metrics
-  delegate   :my_zone, :to => :ext_management_system
```

Comment:
@simon3z why? I added `my_zone` as a method further down in this class. 
You mean if it should be a `virtual_column` or cached somehow?